src/layout/msWorkspace/tilingLayouts/float.ts

```diff
@@ -16,15 +16,16 @@ export class FloatLayout extends BaseTilingLayout {
     static state = { key: 'float' };
     static label = 'Float';
 
-    tileableFocused: any;
+    // TODO: Unused?
+    tileableFocused: Tileable | undefined;
```

Comment:
Used line 37?

Reply:
Yes, but it's never assigned to, as far as I can tell.